python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -190,6 +190,14 @@ class InsertFakesConfig(PipelineTaskConfig,
         default="deep",
     )
 
+    template = pexConfig.Field(
+        doc="If the image is to be used as a template then set to True. This will add only sources with"
```

Comment:
The doc description is a bit confusing and could use some simplifying. Especially the third sentence. As for a suggestion for the names "template" seems fine, but perhaps "image" could be changed to "visistExposure" or "ccdVisitExposure"?

Reply:
The columns themselves might need a bit more descriptive name like `isTemplateSource` or `insertIntoTemplate`. Might help with naming to think of them like this.

Reply:
In case anyone wants to put sources with different fluxes into different coadds, I changed the column names to 'isTemplateSource' and 'isNotOnlyTemplateSource'.

Reply:
I have tried to make this doc clearer and will add more details to the community post instructions when I update them. (Which I will do once we have finalised the discussion on this.)